portfolio/src/main/webapp/index.html

```diff
@@ -92,13 +92,76 @@ <h2>Welcome! Check out what I've been up to.</h2>
           </div>
         </div>
 
-        <div class="section-2">2</div>
-
-        <div class="section-3">3</div>
+        <div class="section-2" id="about">
+          <div class="info">
+            <div class "sub-header">
+            <span class="span-me">About me</span>
+          </div>
 
-        <div class="section-4">4</div>
+          <div class="aboutMe-content">
+            <h2>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+              reprehenderit in voluptate velit esse cillum dolore eu fugiat
+              nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+              sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </h2>
+          </div>
+        </div>
+      </div>
+      <div class="section-3" id="resume">
+        <div class="info">
+          <div class="sub-header">
+            <span class="span-me">Education & Experience</span>
+          </div>
 
-        <div class="section-5">5</div>
+          <div class="resume-content">
+            <ul class="resume-list">
+              <li>
+                <b>Massachusetts Institute of Technology</b>
+                <i>Current</i>
+                <i class="fas fa-book" style="font-size: 24px;" ;></i>
```

Comment:
Looks like "font-size: 24px" is repeated several times throughout. You might consider adding a CSS rule for this.